Purchases/RCBackend.h

```diff
@@ -87,6 +89,12 @@ presentedOfferingIdentifier:(nullable NSString *)offeringIdentifier
                   appUserID:(NSString *)applicationUsername
                  completion:(RCOfferSigningResponseHandler)completion;
 
+- (void)postSubscriberAttributes:(RCSubscriberAttributeDict)subscriberAttributes
+                       appUserID:(NSString *)appUserID
+                      completion:(void (^)(NSError *))completion;
+
+extern NSErrorUserInfoKey const RCShouldMarkSyncedKey;
```

Comment:
Should this go on top of the file? I am not sure what is the most common way of doing this 😄

Reply:
I wasn't really sure what to go with here, happy to move it up 👍